test/__snapshots__/index.test.js.snap

```diff
@@ -20,12 +20,12 @@ basic
 
 
 Build \\"basicLib\\" to dist:
-211 B: basic-lib.js.gz
```

Comment:
the size increase is pretty drastic, do you know what causes it?

Reply:
See the first comment :)

Reply:
@rpetrich made an update to his babel plugin, which improves size quite a bit. On top of that I switched back to `bublé` so that `babel` is only used to transform `async/await` and nothing else. Curiously this all together leads to even smaller size than what's in master 🎉

Reply:
The goal was to move over to Babel completely at some point (#34) to get all of it's transforms, i.e. #172

Reply:
I'm not sure how that would be possible given our size constraint. I tried various ways to get size down after switching to babel and came to the conclusion that it isn't really possible. The generated output by babel is always larger than that of `bublé` and would easily add `+20b` for even our simple fixtures.

Reply:
Hmm :/

Guessing loose mode also doesn't help

Reply:
cc @developit

Reply:
Yup, in loose mode it makes the arguments iteration in the `h()` function larger by rewriting the `arguments` iteration and creating a second copy :/

Reply:
I hadn't thought of using both Bublé and babel - that's actually an ideal solution. We could probably even have Babel apply the class properties transform (most requested feature) but still use Bublé's loose mode for classes!

Reply:
@developit Added the class properties transform plugin 👍